elyra/pipeline/handlers.py

```diff
@@ -47,7 +47,7 @@ async def post(self, *args, **kwargs):
         pipeline_export_path = payload['export_path']
         pipeline_overwrite = payload['overwrite']
 
-        pipeline = PipelineParser().parse(pipeline_definition)
+        pipeline = PipelineParser().parse(pipeline_definition, self.settings['server_root_dir'])
```

Comment:
It might be cleaner to make `root_dir` and attribute (member variable) of `PipelineParser` and convert the two static methods that require `root_dir` as a parameter back to instance methods where the latter of the two can simply reference `self.root_dir`.  This way it doesn't need to be "on the surface" down to where it's used.

Reply:
I was going to ask you about this -- glad you saw! I wanted to get what I had pushed ASAP so that Luciano could test out the functionality, but adding the attribute definitely seems like the way to go